pdx-cs-lbryan/ftpclient
# Installer

See readme for instructions.

How to test:
Use the instructions to set up the installer.
Run the installer then run the program. Make sure the program works.
Uninstall the program.
